system/jlib/jlog.hpp

```diff
@@ -691,6 +691,7 @@ interface jlib_decl ILogMsgFilter : public IInterface
 
 typedef enum
 {
+    LOGFORMAT_undefined,
```

Comment:
I think this ok, but I had  to double check none of these values were serialized - because this changes all the values.